
# HG changeset patch
# User jvos
# Date 1597927682 -7200
# Node ID c0fe72db97a8924c906e4237881aa3c1d5669d19
# Parent  38425dc30b9a25d52fbb0548ea1b0d098888aaed
8252067: Change JavaFX release version in 11-dev to 11.0.9
Reviewed-by: kcr

--- a/build.properties
+++ b/build.properties
@@ -41,7 +41,7 @@
 # UPDATE THE FOLLOWING VALUES FOR A NEW RELEASE
 jfx.release.major.version=11
 jfx.release.minor.version=0
-jfx.release.security.version=8
+jfx.release.security.version=9
 jfx.release.patch.version=0
 
 # Note: The release version is now calculated in build.gradle as the

